build/scripts/build.ps1

```diff
@@ -362,6 +363,60 @@ function Build-InsertionItems() {
     }
 }
 
+function Build-Installer () {
+    #  Copying Artifacts
+    $installerDir = Join-Path $configDir "Installer"
+    if (-not (Test-Path $installerDir)) {
+        New-Item -ItemType Directory -Force -Path $installerDir
+    }
+    
+    $intermidateDirectory = Join-Path $env:TEMP "InstallerTemp"
+    if(Test-Path $intermidateDirectory)
+    {
+        Remove-Item -Path $intermidateDirectory -Recurse -Force
+    }
+    New-Item -ItemType Directory -Force -Path $intermidateDirectory
+
+    ## Copying VsixExpInstaller.exe
+    $vsixExpInstallerDir = Get-PackageDir "RoslynTools.Microsoft.VSIXExpInstaller"
+    $vsixExpInstallerExe = Join-Path $vsixExpInstallerDir "tools\VsixExpInstaller.exe"
+    $vsixExpInstallerExeDestination = Join-Path $intermidateDirectory "tools\vsixexpinstaller"
+    if (-not (Test-Path $vsixExpInstallerExeDestination)) {
+        New-Item -ItemType Directory -Force -Path $vsixExpInstallerExeDestination
+    }
+    Copy-Item -Path $vsixExpInstallerExe -Destination $vsixExpInstallerExeDestination
+    
+    ## Copying VsWhere.exe
+    $vswhere = Join-Path (Ensure-BasicTool "vswhere") "tools\vswhere.exe"
+    $vswhereDestination = Join-Path $intermidateDirectory "tools\vswhere"
+    if (-not (Test-Path $vswhereDestination)) {
+        New-Item -ItemType Directory -Force -Path $vswhereDestination
+    }
+    Copy-Item -Path $vswhere -Destination $vswhereDestination
+
+    ## Copying scripts
+    $installerScriptsFolder = Join-Path $repoDir "src\Setup\InstallerScripts\*.bat"
+    Copy-Item $installerScriptsFolder -Destination $intermidateDirectory -Recurse
+    
+    $installerScriptsFolder = Join-Path $repoDir "src\Setup\InstallerScripts\tools\*.ps1"
+    $intermidatePowershellScriptsDirectory = Join-Path $intermidateDirectory "tools"
+    Copy-Item $installerScriptsFolder -Destination $intermidatePowershellScriptsDirectory -Recurse
+
+    ## Copying VSIXes
+    $vsixDir = Join-Path $configDir "Vsix"
+    $vsixDirDestination = Join-Path $intermidateDirectory "vsix"
+    if (-not (Test-Path $vsixDirDestination)) {
+        New-Item -ItemType Directory -Force -Path $vsixDirDestination
+    }  
+    $RoslynDeploymentVsix = Join-Path $vsixDir "Roslyn\RoslynDeployment.vsix"
+    Copy-Item $RoslynDeploymentVsix -Destination $vsixDirDestination
+    
+    #  Zip Folder
+    $installerZip = Join-Path $installerDir "Roslyn_Preview"
+    $intermidateDirectory = Join-Path $intermidateDirectory "*"
+    Compress-Archive -Path $intermidateDirectory -DestinationPath $installerZip
```

Comment:
Our intent is to begin publishing these on every build. In order to support that I think that we need to be giving these items a versioned name. That way we can line them up with our NuGet packages.